docs/AccessScalarProducts.md

```diff
@@ -0,0 +1,185 @@
+# Access Scalar products on a Kubernetes environment
```

Comment:
I changed "Scalar products" to "ScalarDB and ScalarDL" throughout this document to be more specific. I also incorporated some suggestions from [this comment within the doc](https://github.com/scalar-labs/scalar-kubernetes/pull/196/files#r1128986435).

If either of those are incorrect assumptions, let's discuss!

```suggestion
# Expose ScalarDB or ScalarDL in a Kubernetes cluster environment
```

Reply:
Thank you for incorporating our comments and your suggestions!
The suggested titles look good to me!